pebbles/static/partials/user_dashboard.html

```diff
@@ -1,28 +1,30 @@
 <div id="user-dashboard">
-    <h1>Blueprints</h1>
+    <h2 style="padding-left: 15px;">Resources</h2>
     <div class="row">
-        <div class="alert alert-warning" ng-hide="blueprints != undefined && (blueprints | filter : {is_enabled: 'true'}).length">
+        <div class="alert alert-warning" style= "margin-left: 30px;" ng-hide="blueprints != undefined && (blueprints | filter : {is_enabled: 'true'}).length">
```

Comment:
There's only 1 case of inline style in the whole partials directory according to my grepping and inline styles are a bit of a code smell.

Could you just slap an extra class with a descriptive name to the object and add the class to a CSS file instead?